File: java-main/java-main/Array/Ex2/p2.java
/*WAP to find the number of even and odd integers in a given array of integers*/

import java.util.*;
class Demo{
        public static void main(String[]args){

		Scanner sc = new Scanner (System.in);
		System.out.println("Enter array size:");

		int size=sc.nextInt();
		int arr[]=new int[size];

		int evenCount=0;
		int oddCount=0;

		System.out.println("Enter array elements:");

		for(int i=0;i<arr.length;i++){
                        arr[i]=sc.nextInt();

			if(arr[i]%2==0){
                                evenCount++;
			}else{
				oddCount++;
			}
		}
                System.out.println("Array evevn no count="+ evenCount);
                System.out.println("Array odd no count="+ oddCount);
	}
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/magic2/p2.c
#include<stdio.h>
void main(){

	int size; 
	int count,count1=0;

	printf("Enter array size:");
	scanf("%d",&size);

	int iarr[size];

	printf("Enter array element:");
	for(int i=0;i<size;i++){
		scanf("%d",&iarr[i]);
	}
	

	for(int i=0;i<size;i++){
		if(iarr[i]%2==0){
			count++;

		}else{
			count1++;
		}
	}
	printf("Count of even num in an array is:%d\n",count);

	printf("Count of odd num is an array is :%d\n",count1);
}

